src/index.ts

```diff
@@ -67,20 +68,54 @@ const plugin: JupyterFrontEndPlugin<void> = {
         const entries = document.createElement('div');
         links.map(link => {
           const p = document.createElement('p');
-          const a = document.createElement('a');
-          a.href = link;
-          a.innerText = link;
-          p.appendChild(a);
+          const text: HTMLInputElement = document.createElement('input');
+          text.readOnly = true;
+          text.value = link;
+          text.addEventListener('click', e => {
+            (e.target as HTMLInputElement).select();
+          });
+          text.style.width = '100%';
+          p.appendChild(text);
           entries.appendChild(p);
         });
 
+        // Warn users of the security implications of using this link
+        // FIXME: There *must* be a better way to create HTML
+        const warning = document.createElement('div');
+
+        const warningHeader = document.createElement('h3');
+        warningHeader.innerText = trans.__('Security warning!');
+        warning.appendChild(warningHeader);
+
+        const messages = [
+          'Anyone with this link has full access to your notebook server, including all your files!',
```

Comment:
Since these are user facing strings, it would be great if we could use the `ITranslator` to handle localization.

Reply:
Never mind, it's done below, thanks!

Reply:
yeah, i did them all together!